lib/plugins/features/movies/index.js

```diff
@@ -1,25 +1,38 @@
 'use strict';
 
-const Controller     = require('./controller');
-const MovieValidator = require('../../../validators/movie');
+const Controller         = require('./controller');
+const MovieListValidator = require('../../../validators/movies/list');
+const MovieValidator     = require('../../../validators/movies/create');
```

Comment:
Could we rename this to `MovieCreateValidator` so we have a standard, explicit naming format? I'd maybe even make it `Movies...` to align with the directory format.